### ISSUE TITLE
Not friendly with custom paths.

### ISSUE THREAD

**Issue by mshima** [thymikee/jest-preset-angular#908]
## 🐛 Bug Report

Using jest-preset-angular have unexpected results with webapp path different of `src`.

jest-preset-angular should not provide `src` hardcoded values at moduleNameMapper:
https://github.com/thymikee/jest-preset-angular/blob/a14ab84dac7d40b545bf025049f088f5ba42a134/presets/index.js#L15-L20

Instead, it should provide default values for roots and modulePaths:
```
  roots: ['<rootDir>', '<rootDir>/src'],
  modulePaths: ['<rootDir>/src'],
```

## To Reproduce

- clone https://github.com/mshima/jest-preset-bug-reprodution
Angular application is implemented at `src/main/webapp` instead of `src`.
- npm ci
- npm test

Jest fails with:
```
    Configuration error:

    Could not locate module app/config/input.constants mapped as:
    /Users/mshima/bug-tracker-tsconfig/src/app/$1.

    Please check your configuration for these entries:
    {
      "moduleNameMapper": {
        "/^app\/(.*)$/": "/Users/mshima/bug-tracker-tsconfig/src/app/$1"
      },
      "resolver": undefined
    }
```

Thats because moduleNameMapper takes precedence over modulePaths.

`npm test -- --show-config` (relevant parts):
```
      "moduleNameMapper": [
        [
          "\\.(jpg|jpeg|png)$",
          "/Users/mshima/bug-tracker-tsconfig/node_modules/@angular-builders/jest/dist/jest-config/mock-module.js"
        ],
        [
          "^src/(.*)$",
          "/Users/mshima/bug-tracker-tsconfig/src/$1"
        ],
        [
          "^app/(.*)$",
          "/Users/mshima/bug-tracker-tsconfig/src/app/$1"
        ],
        [
          "^assets/(.*)$",
          "/Users/mshima/bug-tracker-tsconfig/src/assets/$1"
        ],
        [
          "^environments/(.*)$",
          "/Users/mshima/bug-tracker-tsconfig/src/environments/$1"
        ]
      ],
      "modulePathIgnorePatterns": [],
      "modulePaths": [
        "/Users/mshima/bug-tracker-tsconfig/src/main/webapp"
      ],
```

## Expected behavior

Jest should find the relevant modules.

## Error log:

```
# content of error stacktrace :

```

## envinfo

```
System:
    OS:

Npm packages:
    jest:
    jest-preset-angular:
    typescript:
```

**Comment by ahnpnl** [comment]
The configuration in our preset works with basic Angular project generated by Angular CLI. You can always override the configuration on your own for moduleNameMapper. 

I think the best is just removing moduleNameMapper from the preset which makes everyone happy :)

**Comment by ahnpnl** [comment]
Since we are going to have 9.0.0 release soon so that change can be included in 9.0.0 as breaking change.

**Comment by mshima** [comment]
Thanks for the quick answer.

Manually replacing at `node_modules/jest-preset-angular/jest-preset.js`, `moduleNameMapper` with
```
  roots: ['<rootDir>', '<rootDir>/src'],
  modulePaths: ['<rootDir>/src'],
```

The project customs roots and modulePaths applies and `npm test` succeeds.
